agrirouter-api-java-api/src/main/java/com/dke/data/agrirouter/api/service/RevokingService.java

```diff
@@ -1,9 +1,15 @@
 package com.dke.data.agrirouter.api.service.onboard;
 
+import com.dke.data.agrirouter.api.dto.revoke.RevokingError;
 import com.dke.data.agrirouter.api.enums.RevokeResponse;
 import com.dke.data.agrirouter.api.service.parameters.RevokeParameters;
+import java.util.Optional;
 
 public interface RevokingService {
 
   RevokeResponse revoke(RevokeParameters revokeParameters);
+
+  String getLastErrorAsString();
+
+  Optional<RevokingError> getLastError();
```

Comment:
I would prefer `getRevokingError` and `getLastError` to avoid `...asString` within the method name.

Reply:
Done